### PR TITLE
Update the ScholarX 2022 page FAQ section

### LINKED ISSUE #1187: Update the ScholarX 2022 page FAQ section
#### Definition of done: 
The answer for the question "How are mentors assigned?" under FAQ for mentees should be changed to what's given in this issue.

**New content**

> Mentor information and areas of expertise are published on the SEF ScholarX platform. Students will be asked to pick their choice of mentor and will be assigned to a mentor based on their choice as well as the relative strength of their application.

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1187 

## Goals

Updating the ScholarX 2022 page FAQ section

## Approach

I updated the ScholarX 2022 page FAQ section

### Screenshots

Screenshot_20220325-134345-1

  
### Preview Link

https://pr-1188-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
@Gravewalker666 is this ok?

File: scholarx/2022/index.html [hunk 1]
```diff
@@ -464,15 +464,7 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
...
-                                        Mentor information and areas of expertise are
-                                        published on
-                                        the SEF website and social media. Students will be
-                                        asked
-                                        to pick their top three choices in mentors and will
-                                        be
-                                        assigned to selected mentors based on these choices
-                                        as well as the relative strength of their
-                                        application.
+                                        Mentor information and areas of expertise are published on the SEF ScholarX platform. Students will be asked to pick their choice of mentor and will be assigned to a mentor based on their choice as well as the relative strength of their application.
```
Comment (Gravewalker666) [inline scholarx/2022/index.html]: Let's break this into few separate lines. This exceeds the max character limit.
Looks good otherwise. :)

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: scholarx/2022/index.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: Sure!

**Comment by YoshithaRathnayake** [comment]
@Gravewalker666 Now is it ok?

File: scholarx/2022/index.html (hunk 1 above)
Comment (YoshithaRathnayake) [reply scholarx/2022/index.html]: @Gravewalker666 Now is it ok?

**Comment by YoshithaRathnayake** [comment]
I breaked that text into few lines like the old one.

**Review by Gravewalker666 — APPROVED** [review]
Thank you @YoshithaRathnayake

**Pull request merged**